translations/en/global_indicators.yml

```diff
@@ -358,20 +361,13 @@
 4-6-1-title: Proportion of population in a given age group achieving at least a fixed
   level of proficiency in functional (a) literacy and (b) numeracy skills, by sex
 4-7-1-title: Extent to which (i) global citizenship education and (ii) education for
-  sustainable development, including gender equality and human rights, are mainstreamed
-  at all levels in (a) national education policies; (b) curricula; (c) teacher education;
-  and (d) student assessment
-4-a-1-title: Proportion of schools with access to (a) electricity; (b) the Internet
-  for pedagogical purposes; (c) computers for pedagogical purposes; (d) adapted infrastructure
-  and materials for students with disabilities; (e) basic drinking water; (f) single-sex
-  basic sanitation facilities; and (g) basic handwashing facilities (as per the WASH
-  indicator definitions)
+  sustainable development are mainstreamed in (a) national education policies; (b)
+  curricula; (c) teacher education; and (d) student assessment
+4-a-1-title: Proportion of schools offering basic services, by type of service
 4-b-1-title: Volume of official development assistance flows for scholarships by sector
   and type of study
-4-c-1-title: 'Proportion of teachers in: (a) pre-primary; (b) primary; (c) lower secondary;
-  and (d) upper secondary education who have received at least the minimum organized
-  teacher training (e.g. pedagogical training) pre-service or in-service required
-  for teaching at the relevant level in a given country'
+4-c-1-title: Proportion of teachers with the minimum required qualifications, by education
```

Comment:
I've just looked at Excel file - this is actually:
Proportion of teachers with the minimum required qualifications, by education level<sup>i</sup>
where i is a footnote:

> Refinement of the indicator name approved by the Inter-agency and Expert Group on SDG Indicators (IAEG-SDGs) on 13 March and 2 April 2020. Final approval pending the 52nd session of the Statistical Commission in March 2021.

There are also a number (7) of other footnotes which may mean there are other instances where they may be a digit added on the end.

@brockfanning is there anything that could be done to resolve this?